tools/msconvert/msconvert_macros.xml

```diff
@@ -25,7 +25,9 @@
     #else
       --input=${input}
       #if hasattr($input, 'display_name')
-      --input_name='${input.display_name}'
+      ##--input_name='${input.display_name}'
+      #set basename = $input.display_name
```

Comment:
using display_name seems always be dangerous to me. Can we not hardcode this to "foo.bar"?